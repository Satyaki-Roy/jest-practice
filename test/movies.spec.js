import { beforeEach, describe, test } from "@jest/globals";
import movies from "../src/movies";

describe("movies", () => {
  let myMovies = [];

  beforeEach(() => {
    myMovies = [
      {
        title: "God Father",
        rate: null,
      },
    ];
  });

  test("add", () => {
    movies.add(myMovies, "Batman");
    expect(myMovies).toMatchSnapshot();
  });

  test("rate", () => {
    movies.rate(myMovies[0], 5);
    expect(myMovies).toMatchSnapshot();
  });
});
