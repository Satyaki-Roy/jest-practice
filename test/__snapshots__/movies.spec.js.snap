// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`movies add 1`] = `
Array [
  Object {
    "rate": null,
    "title": "God Father",
  },
  Object {
    "rate": null,
    "title": "Batman",
  },
]
`;

exports[`movies rate 1`] = `
Array [
  Object {
    "rate": 5,
    "title": "God Father",
  },
]
`;
